### DISCUSSION TITLE
Chocolately AltSnap Outdated! ~ Currently version 1.62

### DISCUSSION THREAD

**Discussion by dillacorn** [RamonUnch/AltDrag#570] [General]
### since we're soon to have version 1.64 after per-release is completely ironed out for full release, can someone take over package maintainer since it seems he has gone inactive?

**Comment by RamonUnch** [comment]
I will try again to update it but last time I tried it got rejected by automated validation for a reason I did not get and had little time to investigate.
I will try and if you have the Energy I can give you the choco package.

**Reply by dillacorn** [reply]
my mistake on package maintainer.. I literally didn't read the name and thought someone else was maintaining the package not the main dev.. 

maybe I will just stick with git if you're getting false positives through automated choco validations... bet that's frustrating..

**Reply by RamonUnch** [reply]
I still get a reject and I cannot understand the log.
their test system fails to install altsnap and I do not see how.
It works for all the machines I tested. Even Win7 32 bits. and Win11 x64

Can you try on your system?
`choco install altsnap --version=1.64`
I have no clue.
Error log from them: https://gist.github.com/choco-bot/849cd31accdd559b2070ee32e3ca6366

**Reply by dillacorn** [reply]
`choco install altsnap --version=1.64` does not work. Package not found

tried `choco install altsnap --version=1.64 --debug --force` but receiving the same error..

May be some sort of issue on there testing environment because it's stating there is an issue installing possibly due to the current install not being a completely silent install.. aka pop-ups that may have not been present before in previous releases... 

Some applications aren't present on Chocolatey and are only available on scoop... maybe you should just move on and de-list AltSnap on Chocolatey and continue it on Scoop like you currently are.

**Comment by RamonUnch** [comment]
should be fixed for now and in the future.
